src/cli/playbook/mod.rs

```diff
@@ -1,5 +1,5 @@
 mod input;
-mod ipa;
+pub mod ipa;
```

Comment:
I think we can keep it private and re-export the malicious function like this

```rust
pub use self::ipa::malicious;
```

or if we can make a single function like suggested above, we won't even need that

Reply:
yeah, was able to just remove after the change above